mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -2113,6 +2139,52 @@ impl ElectrumClient {
                 }),
         )
     }
+
+    pub(crate) fn get_servers_with_latest_block_count(&self) -> UtxoRpcFut<(Vec<String>, u64)> {
+        let ticker = self.coin_ticker.clone();
+        let selfi = self.clone();
+        let fut = async move {
+            let mut futures = vec![];
+            let connections = selfi.connections.lock().await;
+            for connection in connections.iter() {
+                let addr = connection.addr.clone();
```

Comment:
Initialising `Vector` here with capacity is really worth for performance wise

```rust
 let connections = selfi.connections.lock().await;
 let mut futures = Vec::with_capacity(connections.len());
```

Reply:
how about the below instead?
```rust
let futures = connections
    .iter()
    .map(|connection| {
        let addr = connection.addr.clone();
        selfi
            .get_block_count_from(&addr)
            .map(|response| (addr, response))
            .compat()
    })
    .collect::<Vec<_>>();
```

Reply:
Done